testah-junit/src/main/java/org/testah/runner/http/load/HttpAkkaStats.java

```diff
@@ -137,7 +136,12 @@ public Long getLongestDuration() {
      * @return the list of all durations
      */
     public List<Long> getDurations() {
-        return Doubles.asList(statsDuration.getValues()).stream().map(val -> new Double(val).longValue()).collect(Collectors.toList());
+        List<Long> longs = new ArrayList<>();
```

Comment:
I tried 
{code}
    @Test
    public void test() {
        double[] doubles = {1.0, 2.0, 3.0};
        Doubles.asList(doubles).stream().map(val -> val.longValue()).collect(Collectors.toList());
    }
{code}
which worked fine (at least out of IntelliJ), not sure why 
{code}
    public List<Long> getDurations() {
        return Doubles.asList(statsDuration.getValues()).stream().map(val -> val.longValue()).collect(Collectors.toList());
{code}
should be a problem

Reply:
its a spot bug issue with unboxing, its converting from double to Double to long to Long, it was flagged as an issue that's the only reason I changed it, I did think to suppress the warning, but opted to change it to not get flagged.  https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bx-boxed-value-is-unboxed-and-then-immediately-reboxed-bx-unboxing-immediately-reboxed